src/compiler/repositories.md

```diff
@@ -0,0 +1,80 @@
+- A fork of [Enzyme], a high-performance automatic differentiator of LLVM and MLIR (more info at this [link][enzyme-mit]). This fork is maintained by [ZuseZ4].
+- [`thorin`][thorin], a DWARF packaging utility supporting GNU extension and DWARF 5 package formats. Primarily maintained by [davidtwco][davidtwco].
+- The [Rust Forge], the documentation website you're reading. Co-maintained by the Rust project collectively.
+- [`ar_archive_writer`][ar_archive_writer]: Like the other `LLVM::APFloat` library, its license is slightly different than our other repositories, see [ar_archive_writer#licensing].
+- [`datafrog`][datafrog], a lightweight [Datalog] engine intended to be embedded in other Rust programs (TODO: status?)
+- [`ena`][ena] is an implementation of union-find / congruence-closure in Rust (TODO: clarify what this is)
+- [`literal-escaper`][literal-escaper] is a library to unescape string literals. It is used by [`rustc_lexer`][rustc_lexer] and [`proc_macro`][proc_macro].
+- [`miri`][miri] is the interpreter for Rust's mid-level intermediate representation. Detects unsafe code that fails to uphold its safety requirements.
+- [`measureme`][measureme] is a library for recording and serializing `rustc` events to a binary format. Currently only for internal use within `rustc` itself.
+- [`odht`][odht] is a crate for hash tables that can be mapped from disk into memory without up-front decoding. Currently only for internal use within `rustc` itself (TODO: status?)
+- [`rustc-demangle`][rustc-demangle]: Demangling for `rustc` symbols ([documentation][rustc_demangle_docs]).
+- [`rustc-hash`][rustc-hash] is a non-cryptographic hashing algorithm used by `rustc`
+- [`rustc-rayon`][rustc-rayon] is a fork of the Rayon data parallelism library for Rust. This is part of an ongoing effort to parallelize the `rustc` compilation, see our [working areas].
+- [`rustc-stable-hash`][rustc-stable-hash] is a cross-platform, deterministic and *not secure* hashing algorithm used by `rustc`.
+- [`stacker`][stacker] a library to help grow the stack when it runs out of space, see [documentation][stacker_docs].
+
+Other repositories with tools for internal use:
+
+- [`cargo-bisect`][cargo-bisect]: a tool to bisect [regressions] in the rust compiler, very useful to find where a bug was introduced.
+- We have a [calendar] where we all teams register their meetings. Calendar clients can pull the `.ics` files and receive updates.
+- [`jobserver-rs`][jobserver-rs]: an implementation of the GNU Make jobserver for Rust, see [documentation][jobserver_docs].
+- [`josh-sync`][josh-sync] : a library to perform [`Just One Single History`][josh] synchronizations (pull and push) of Josh subtrees in the rust-lang/rust repository.
+
+[Here the full list][other] of all the repositories under purview of the Compiler Team.
```

Comment:
Remark: this is not *quite* accurate, I think this is just what that repos the compiler github team has write access to (?) I'd maybe just leave this off.